velox/exec/tests/HashJoinTest.cpp

```diff
@@ -1217,7 +1217,8 @@ TEST_F(HashJoinTest, nullAwareAntiJoinWithFilterAndNullKey) {
   createDuckDbTable("t", {leftVectors});
   createDuckDbTable("u", {rightVectors});
   auto planNodeIdGenerator = std::make_shared<PlanNodeIdGenerator>();
-  for (const std::string& filter : {"u1 > t1", "u1 * t1 > 0"}) {
+  std::vector<std::string> filters = {"u1 > t1", "u1 * t1 > 0"};
+  for (const std::string& filter : filters) {
```

Comment:
Looks like a bug in compiler?

Reply:
I am curious if the following code works or not?
```
for (const std::string& filter : std::vector<std::string>({"u1 > t1", "u1 * t1 > 0"})) {
```